omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -165,13 +169,25 @@ public DelegatingResourceDescription getUpdatableProperties() throws ResourceDoe
 	 * @return all orders for a given patient (possibly filtered by context.type)
 	 */
 	@Override
-	protected PageableResult doSearch(RequestContext context) {
+	protected PageableResult doSearch(RequestContext context) throws ResponseException {
 		String patientUuid = context.getRequest().getParameter("patient");
+		String stat = context.getRequest().getParameter("status");
+		if ((StringUtils.isNotEmpty(patientUuid)) && StringUtils.equals(stat, "any")) {
+             try {
+            	 throw new MissingRequiredIdentifierException("Sorry You did not Specify a Patient #UUID");
+			}
+			catch (MissingRequiredIdentifierException e) {
```

Comment:
Why are we catching this?  I believe we just want to throw the exception in this case.